### PR TITLE
remove scorecard gh action

### PR DESCRIPTION
**Pull request by KPostOffice**
# Issue link
closes: #365 

# What changes have been made
removed the scorecard gh action from the `.workflows` repo

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: .github/workflows/olm_scorecard.yml [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-      run: |
-        sudo wget -O /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.6.1/yq_linux_amd64
-        sudo chmod +x /usr/bin/yq
-
-    - name: Modify scorecard config
-      run: |
-        yq e 'del(.stages[].tests[] | select(.labels.test == "olm-crds-have-resources-test"))' -i bundle/tests/scorecard/config.yaml
-
-    - name: Run Operator SDK Scorecard
-      run: make scorecard-bundle
```
Comment (tedhtchang) [inline .github/workflows/olm_scorecard.yml:28]: I think we may remove the scorecard-bundle in the Makefile.

Comment (KPostOffice) [reply .github/workflows/olm_scorecard.yml:28]: Removed

**Review by jbusche — APPROVED** [review]
LGTM

**Pull request merged**